ZelBack/src/services/appsService.js

```diff
@@ -8986,8 +8986,32 @@ async function trySpawningGlobalApplication() {
 /**
  * To check and notify peers of running apps. Checks if apps are installed, stopped or running.
  */
+let nodeConfirmedOnLastCheck = true;
 async function checkAndNotifyPeersOfRunningApps() {
   try {
+    const isNodeConfirmed = await generalService.isNodeStatusConfirmed();
```

Comment:
I disagree with removal of applications from nodes when nodes are no longer on the network.
Those applications should continue to be running. If we have some flux network issues, the apps will still be running, not affected by flux network issues and fdm remembers these old ips.